### DISCUSSION TITLE
Import eventbridge resources

### DISCUSSION THREAD

**Discussion by cs31415** [GoogleCloudPlatform/terraformer#1241] [Ideas]
Any plans to add support for importing eventbridge rules? I have tried using `--resources aws_cloudwatch_event_rule`, but it doesn't import eventbridge rules.

**Comment by TheWesDias** [comment]
Hi @cs31415 I'm addressing this issue here in this PR https://github.com/GoogleCloudPlatform/terraformer/pull/2082
